modules/msp/apm/trace/trace.service.go

```diff
@@ -67,6 +67,11 @@ func (s *traceService) getDebugStatus(lang i18n.LanguageCodes, statusCode debug.
 	}
 }
 
+// TODO @innerpeacez add GetSpanDashboards()
```

Comment:
what's this

Reply:
@innerpeacez  
proto-go version cause CI-check failed